src/routes/auth.js

```diff
@@ -14,21 +14,32 @@ const {
 
 const router = express.Router()
 
+const transformUserResponse = ({accessToken,
```

Comment:
simply take in `user` or `userDetails` as a param and do the transformation

Reply:
yes please